lib/components/conversation/conversation_item.dart

```diff
@@ -0,0 +1,168 @@
+    if (_choosen) {
+      renderElement.classes.add("conversation-item--selected");
+    }
+
+    var checkboxWrapper = DivElement()..className = "conversation-item-checkbox--wrapper";
+    _checkboxElement = CheckboxInputElement()
+      ..onInput.listen((event) {
+        event.stopPropagation();
+        event.preventDefault();
+        var checked = (event.currentTarget as CheckboxInputElement).checked;
+        if (checked) {
+          if (_onSelectController.hasListener) {
+            _onSelectController.sink.add(_id);
+          } else {
+            logger.warning("No listener for ConversationItemView.onSelect");
+          }
+        } else {
+          if (_onDeselectController.hasListener) {
+            _onDeselectController.sink.add(_id);
+          } else {
+            logger.warning("No listener for ConversationItemView.onSelect");
+          }
+        }
+      });
+    if (_selected) {
+      _checkboxElement.checked = true;
+    }
+    checkboxWrapper.append(_checkboxElement);
+
+    var contentWrapper = DivElement()
+      ..className = "conversation-item-content--wrapper"
+      ..onClick.listen((event) {
+        if (_onChooseController.hasListener) {
+          _onChooseController.sink.add(_id);
+        } else {
+          logger.warning("No listener for ConversationItemView.onCHoose");
+        }
+      });
+
+    var idElement = DivElement()
+      ..className = "conversation-item-id"
+      ..innerText = _id;
+
+    var messageElement = DivElement()..className = "conversation-item-message";
+    var messageTextElement = SpanElement()
+      ..className = "conversation-item-message-text"
+      ..innerText = _message;
+    _messageStatusElement = SpanElement()..className = "conversation-item-status";
+    _updateStatus(_status);
+
+    messageElement..append(messageTextElement)..append(_messageStatusElement);
+    contentWrapper..append(idElement)..append(messageElement);
+    renderElement..append(checkboxWrapper)..append(contentWrapper);
+
+    this._onChooseController = StreamController();
+    this._onChoose = _onChooseController.stream;
+    this._onSelectController = StreamController();
+    this._onSelect = _onSelectController.stream;
+    this._onDeselectController = StreamController();
+    this._onDeselect = _onDeselectController.stream;
+  }
+
+  void _updateStatus(ConversationItemStatus status) {
+    var messageStatusClasses = _messageStatusElement.classes
+      ..removeWhere((classname) {
+        return classname.indexOf("converversation-item-status--") == 0;
+      });
+    var renderElementClasses = renderElement.classes
+      ..removeWhere((classname) {
+        return classname.indexOf("conversation-item--selected") != 0 && classname.indexOf("conversation-item--") == 0;
+      });
+    switch (status) {
+      case ConversationItemStatus.draft:
+        renderElementClasses.add("conversation-item--draft");
+        renderElement.className = renderElementClasses.join(" ");
```

Comment:
I think best to use the in-build functionality for managing element classes as lists, rather than strings, so e.g. with the above in mind, this would become:
```suggestion
        renderElement.classes.add("conversation-item--draft");
```